internal/storage/binlog_util.go

```diff
@@ -1,12 +1,18 @@
 package storage
 
 import (
+	"fmt"
 	"strconv"
 	"strings"
 )
 
+// ParseSegmentIDByBinlog parse segment id from binlog paths
+// if path format is not expected, returns error
 func ParseSegmentIDByBinlog(path string) (UniqueID, error) {
-	// binlog path should consist of "files/insertLog/collID/partID/segID/fieldID/fileName"
+	// binlog path should consist of "[prefix]/insertLog/collID/partID/segID/fieldID/fileName"
 	keyStr := strings.Split(path, "/")
+	if len(keyStr) != 7 {
```

Comment:
Maybe changing to >= 7 is better? Because we can't decide what the user defines the `prefix` as.

Reply:
Yes， we don't limit the prefixes to use x/x/x or no x/  within etcd configurations.  it should not support  x/x/... as a prefix if the length is 7

Reply:
currently binlog path format is fixed. When collector scans the oss directory, if the path does not matches the format we define, we shall return an error to ignore this file.

Reply:
Think twice, if prefix contains "/", the segment fetch logic cannot work

Reply:
more prechecking of configuration should be implemented during the parameters initialize